### ISSUE TITLE
Add binaries to releases

### ISSUE THREAD

**Issue by makeworld-the-better-one** [MasterQ32/kristall#19]
It'd be nice to see! I have no idea how easy it would be across multiple platforms though.

**Comment by MasterQ32** [comment]
Yes, this is planned for a future point, right now i want to scrub more features before doing the automated crossplatform build walk again
